Automattic/wp-calypso
# Paid Stats: Add expandable feature list to upsell

Related to https://github.com/Automattic/jetpack-roadmap/issues/2056

## Proposed Changes

* Update design of Upsell to match new design
* Update the feature line for Stats at checkout
* Fix stats chart lines when navigating from home to stats

![image](https://github.com/user-attachments/assets/9a375f6c-cc7d-4160-8ea9-155b1525a33f)

## Testing Instructions



* Navigate to a free site created after Dec 11 in the live branch
* Compare the Upsell to the video in the comments below
* Check that checkout features properly display 'Detailed traffic stats beyond the last 7 days and site insights'
* Check that stats chart is not break out of it's boundaries when navigating from Home to Stats page.

## Pre-merge Checklist
